client/js/components/procedure/admin/DpAddOrganisationList.vue

```diff
@@ -147,11 +230,57 @@ export default {
         .catch(() => {
           dplan.notify.error(Translator.trans('warning.invitable_institution.not.added'))
         })
+    },
+
+    getInstitutionsWithContacts () {
+      const invitableToebFields = [
+        'legalName',
+        'participationFeedbackEmailAddress',
+        'locationContacts'
+      ]
+
+      const locationContactFields = ['street', 'postalcode', 'city']
```

Comment:
since these arrays are static I would move them to data() to enable a simplification for this piece of code.

Reply:
right, done here https://github.com/demos-europe/demosplan-core/pull/2630/commits/9f3492dafac06540bd10c8a26ca6c52493ee430c